components/form/orientation.md

```diff
@@ -10,13 +10,18 @@ position: 10
 
 # Form Orientation
 
+
 The Form component for Blazor allows you control its orientation through the `Orientation` parameter. It takes a member of the `Telerik.Blazor.Enums.FormOrientation` enum:
 
 * `Horizontal`
 
 * `Vertical` - by default the Form will be vertical.
 
->caption Change the orientation of the Form
+Setting the orientation to `Horizonal` will place the labels to the right hand-side of the editors and thus reduce the vertical space taken by the Form.
```

Comment:
Move these explanations to the list above so they are next to the corresponding values

Reply:
Done